### PR TITLE
docs: playground

### PR DESCRIPTION
**Pull request by m0ksem**
closes https://github.com/epicmaxco/vuestic-ui/issues/4018
closes https://github.com/epicmaxco/vuestic-ui/issues/3702
closes https://github.com/epicmaxco/vuestic-ui/issues/3162

Removed broken Codesandbox and added playground on ui.vuestic.dev.

New button: (not sure about icon tho)
image

Playground for component example looks like a modal:
> http://localhost:3000/ui-elements/button
image

After user changed something in playground, it can be shared via url. Url looks like:
> http://localhost:3000/play
image

---

I use vue-repl here to make a sandbox. We need a complete ES build in one file for vue-repl, so I build one in nuxt module before docs build start. Vue-repl makes it easy to share state between pages using serialization. So we don't need to store user's changes on backend - it is stored in url. 
When user open playground from code example in modal url changes to ui.vuestic.dev/play#state so it can be easily shared. When modal is closed, url changes back to docs page.

I believe this will help users to make reproduction easier.

Sandbox supports tailwind, but will not allow installing additional dependencies, what I think OK.

---

This PR also closes https://github.com/epicmaxco/vuestic-ui/issues/4019, I found out this problem when tried to use our es module from CDN.

---

Downside of this PR is that for now, it is impossible to test not latest vuestic version. In such case, I think user still able to make codesandbox reproduction.

### CODE REVIEW

**Comment by Roman4437** [comment]
There's an overflow issue 
image

**Comment by Roman4437** [comment]
during focus or selection, a default outline appears

**Comment by Roman4437** [comment]
The background is static and doesn't change with the theme. Also, white background doesn't look good with elements that have dropdowns

**Comment by Roman4437** [comment]
Some components are not working properly, such as the date-picker or table

**Comment by Roman4437** [comment]
tailwind classes works unless they are custom or not defined through the docs

File: packages/docs/.gitignore [hunk 1]
```diff
@@ -7,3 +7,4 @@ node_modules
 .env
```
Comment (asvae) [inline packages/docs/.gitignore:7]: Got an error on `serve:docs`.

On `develop` docs works fine.

image

Comment (m0ksem) [reply packages/docs/.gitignore:7]: Updating deps and clearing cache might help.

Comment (Roman4437) [reply packages/docs/.gitignore:7]: yes, i had the same error and have fixed by clearing cookies

**Comment by m0ksem** [comment]
> tailwind classes works unless they are custom or not defined through the docs

I updated PR, might be fixed. Can you share an example where the issue happens? Thanks.

**Comment by Roman4437** [comment]
> > tailwind classes works unless they are custom or not defined through the docs
> 
> I updated PR, might be fixed. Can you share an example where the issue happens? Thanks.

It's working properly now

File: packages/docs/pages/play.vue [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+      <Play
+        v-model:state="sandboxState"
+        class="h-[100%] flex-1"
+        :code="makeCode(scriptText, templateText, '')"
+      />
+    </ClientOnly>
+
+    <div class="flex justify-between items-center py-2 px-4 border-t-2 border-[var(--va-background-border)] bg-[var(--va-background-secondary)]">
+      <div>
+        Playground <span class="text-[var(--va-secondary)]">vuestic-ui@latest - vue@latest</span>
```
Comment (m0ksem) [inline packages/docs/pages/play.vue:70]:
```suggestion
        Playground <span class="text-[var(--va-secondary)]">vuestic-ui@latest - vue@latest - tailwindcss@latest</span>
```

**Comment by m0ksem** [comment]
Looks like `<style lang="*"` isn't supported yet by vue-repl.
image


We also need to add config, similar to codesandbox's config, so it can load additional libs, like AGGrid.

File: packages/docs/components/Play.client.vue
Comment (asvae) [inline packages/docs/components/Play.client.vue:1]: Mobile version for modal doesn't work too well (full-screen works fine).

image

I'm thinking we need 2 buttons here:
image copy
image close

Sandbox link I don't think is useful for anything.
I would also suggest not to use primary color for close.

image

**Comment by asvae** [comment]
For Asva to finish:

Master → 1.8.4

Develop → compiled (npm)

Experimental → compiled (npm)

Master → [1.8.3, 1.8.4]

Develop → [1.8.3, 1.8.4, 1234567a, 123567b]

play - source code

play - vuestic-version

**Comment by m0ksem** [comment]
> For Asva to finish:
> 
> Master → 1.8.4
> 
> Develop → compiled (npm)
> 
> Experimental → compiled (npm)
> 
> Master → [1.8.3, 1.8.4]
> 
> Develop → [1.8.3, 1.8.4, 1234567a, 123567b]
> 
> play - source code
> 
> play - vuestic-version
> 
> * Some examples still use old codesandbox - we need to cover that.

I have a PR for build tool. Let's deal with it later, when my PR is merged. It will be easy to add a esm-browser build, which we could use from npm.

File: packages/docs/components/Play.client.vue
Comment (m0ksem) [reply packages/docs/components/Play.client.vue:1]: I think it is a problem caused by VaModal... Moved that to @VasylHryha.

**Comment by asvae** [comment]
I moved everything we discussed to [this issue](https://github.com/epicmaxco/vuestic-ui/issues/4078). So we can merge this PR.

**Pull request merged**